deployment_scripts/appimage/README.md

```diff
@@ -0,0 +1,37 @@
+# Monkey Island AppImage
+
+## About
+
+This directory contains the necessary artifacts for building a prototype
+monkey_island AppImage using appimage-builder.
+
+## Building an AppImage
+
+1. Create a clean VM or LXC (not docker!) based on Ubuntu 18.04.
+1. Update the OS with `sudo apt update && sudo apt upgrade`
```

Comment:
Any reason why this shouldn't be a part of build script?

Reply:
Arguably yes, but in its current form, no. I'll add it to the build script. We can revisit it later.